src/themes/mainTheme.ts

```diff
@@ -205,9 +206,9 @@ export const mainTheme = createTheme(
 
 export const alternatingTableRowColor: SxProps = {
   tr: {
-    background: Color.AlternativeBackground,
+    bgcolor: 'secondary.main',
```

Comment:
oh, man kan bruke paletten direkte her ha, lurt!

Reply:
betyr det at vi kan gjøre det for farge på link underline, og background paper også?